### ISSUE TITLE
Anything you'd like to share?

### ISSUE THREAD

**Issue by yucc-leon** [SUSTech-IDEA/SUS-Chat#1]
Congratulations on your amazing openllm leaderboard ranking!
I'm very curious about the technical details in this work and totally agree with your _do right instruction tuning_ title. So would there be any publication or open-source data soon?

**Comment by fecet** [comment]
Yes, we are in the process of compiling comprehensive details for our forthcoming paper, which will include both the training code and the associated data.

**Comment by yucc-leon** [comment]
> Yes, we are in the process of compiling comprehensive details for our forthcoming paper, which will include both the training code and the associated data.

Looking forward to these releases and I'll close this issue after that.
